charts/gardener-extension-provider-aws/values.yaml

```diff
@@ -10,10 +10,10 @@ maxSurge: 50%
 resources: {}
 vpa:
   enabled: true
-# resourcePolicy:
```

Comment:
@MartinWeindel , these are the VPA values for the extension itself. If you want to adapt the values for the CCM (as stated in the PR description), this would be the right place - https://github.com/gardener/gardener-extension-provider-aws/blob/master/charts/internal/seed-controlplane/charts/cloud-controller-manager/templates/vpa.yaml.

Reply:
It was too busy yesterday, thanks. I will fix it.